From 2de77393e64207dfbef0a4eb4dbd09c75e4d88fe Mon Sep 17 00:00:00 2001
From: Mohanram Meenakshisundaram <mohanram.meenakshisundaram@intel.com>
Date: Thu, 20 Feb 2020 16:35:56 +0530
Subject: [PATCH] CHROMIUM: intel_idle: Add Jasperlake support

This adds JASPERLAKE CPU support to intel idle driver.

BUG=b:149808941
TEST=Build and DUT boot with 5.4 kernel and check intel_idle driver loaded "cat /sys/devices/system/cpu/cpuidle/current_driver"

Change-Id: I490e8b0a7fc61491f3aa4cd222b623d6c7619d68
Signed-off-by: Mohanram Meenakshisundaram <mohanram.meenakshisundaram@intel.com>
---
 drivers/idle/intel_idle.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/idle/intel_idle.c b/drivers/idle/intel_idle.c
index a5d6adc5b399..1937128632e7 100644
--- a/drivers/idle/intel_idle.c
+++ b/drivers/idle/intel_idle.c
@@ -1094,6 +1094,7 @@ static const struct x86_cpu_id intel_idle_ids[] __initconst = {
 	INTEL_CPU_FAM6(ATOM_GOLDMONT_PLUS,	idle_cpu_bxt),
 	INTEL_CPU_FAM6(ATOM_GOLDMONT_D,		idle_cpu_dnv),
 	INTEL_CPU_FAM6(ATOM_TREMONT_D,		idle_cpu_dnv),
+	INTEL_CPU_FAM6(ATOM_TREMONT_L,          idle_cpu_bxt),
 	{}
 };
 
-- 
2.25.0.265.gbab2e86ba0-goog

